db/seeds.rb

```diff
@@ -59,6 +61,23 @@
 bridge_builders2 = Tenant.find(2)
 bridge_builders3 = Tenant.find(3)
 
+
+# user as a tenant
+tenant_user = User.create!(:username => "Jwan622",
+             :first_name => "Jeff",
+             :last_name => "Wan",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.